packages/php-wasm/compile/Dockerfile

```
@@ -1055,5 +1101,12 @@ RUN \
         rm /root/output/php.js && \
         \
         # Rename the build files to their final names
-        mv /root/output/php-module.js "/root/output/$JS_FILENAME" && \
-        mv /root/output/php.wasm "/root/output/$WASM_FILENAME"
\ No newline at end of file
+        mv /root/output/php-module.js "/root/output/$JS_FILENAME"; \
+        mkdir -p /root/output/${PHP_VERSION_ESCAPED}/; \
+		mv /root/output/php.wasm "/root/output/${PHP_VERSION_ESCAPED}/$WASM_FILENAME"; \
+		if [ "${WITH_SOURCEMAPS}" = "yes" ]; then \
+			sed -i 's/..\/php-src/php-src/g' /root/output/php.wasm.map; \
```

Unescaped `.` stands for any character, so `../../root/php-src` gets replaced with `../../rophp-src`. 

The following regexp does the trick:

```suggestion
            sed -i 's#"\.\./\.\./root/#"#g' /root/output/php.wasm.map
```

We could [avoid this replacement entirely](https://github.com/emscripten-core/emscripten/blob/e18753e6fbbf2d690e5a750165c693bb6bfec122/tools/building.py#L1162) by building to `/root`, but that would require more changes to the pipeline so let's leave it for now.